pycrostates/cluster/_base.py

```diff
@@ -560,7 +558,7 @@ def save(self, fname: Union[str, Path]):
         self._check_fit()
         _check_type(fname, ("path-like",), "fname")
 
-    @verbose_decorator
+    @verbose
```

Comment:
I don't know why in this file and only this file the decorator was renamed. I'll see if a test break, but did I miss something?